packages/patternfly-4/react-table/src/components/Table/ExpandableRowContent.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@patternfly/react-styles';
-import { tableExpandableRowContent } from '@patternfly/patternfly-next/components/Table/table.css';
+import { tableExpandableRowContent } from '@patternfly/patternfly/components/Table/table.css';
```

Comment:
Please wait for https://github.com/patternfly/patternfly-react/pull/1314 and rebase.

Reply:
Updated with your changes @karelhala